charts/home-assistant/1.3.0/questions.yaml

```diff
@@ -75,6 +75,30 @@ questions:
       default: "Etc/UTC"
       $ref:
         - "definitions/timezone"
+
+  - variable: PUID
+    group: "Configuration"
+    label: "PUID"
+    description: "The UserID of the user running the application and owning the files"
+    schema:
+      type: int
+      default: 568
+
+  - variable: PGID
+    group: "Configuration"
+    label: "PGID"
+    description: "The groupID of the user/group running the application and owning the files"
+    schema:
+      type: int
+      default: 568
+
+  - variable: UMASK
+    group: "Configuration"
+    label: "UMASK (advanced)"
+    description: "The UMASK used if supported by the application"
+    schema:
+      type: string
+      default: "002"
```

Comment:
Home assistant does not run as another user --  It only works as root

Reply:
Only the virtualenv install can run as non-root user

Reply:
You don't seem to understand what this does. It only does two things:
1. Set the PUID and PGID variables which **ONLY** get used by containers using s6_overlay (or made compatible with it)
2. It is used for setting storage permissions, if enabled on a per-storage basis

However: PUID and PGID are ALWAYS set, hardcoded in common. So removing this does not do anything.
I also make it mandatory, because I've future use for these values on all charts.

Reply:
I've looked into it.
HomeAssistant does not support "Run as Non-root", but should be using s6_overlay since version 0.107 (or something like that).
Run-as-non-root, is not the same as s6_overlay, s6_overlay changes the permissions after start but requires* root to start correctly.
run-as-non-root would be done on helm/k8s using securityContext. Using PGID and PUID for setting the securityContext, is on my to-do list, but will always be optional. primarily because s6_overlay doesn't respond well on setting those.

* Not 100% true, the s6_overlay time has been working on some workarounds but using those is hit-or-nis

Reply:
If this happens to generate errors, please report it and/or set the default PUID/PGID to `0`